.github/workflows/docker-server.yml

```diff
@@ -55,6 +55,21 @@ jobs:
           build-args: |
             GITHUB_SHA_SHORT=${{ steps.build_args.outputs.github_sha_short }}
 
+      - name: Dispatch Commit image CICD
+        if: github.event_name == 'push'
+        env:
+          PAT: ${{ secrets.COMMANDER_DATA_TOKEN }}
+          PARENT_REPO: temporalio/cicd-builds
+          PARENT_BRANCH: master
+          WORKFLOW_ID: publish-stage-results.yml
+          STAGE: builds
+          COMPONENT: temporal
+          TAG: sha-${{ steps.build_args.outputs.github_sha_short }}
+          AUTHOR: ${{ github.event.pusher.name }}
+          COMMIT_MESSAGE: ${{ github.event.head_commit.message }}
+        run: |
+          curl -fL -X POST -H "Accept: application/vnd.github.v3+json" -H "Authorization: token ${{ env.PAT }}" https://api.github.com/repos/${{ env.PARENT_REPO }}/actions/workflows/${{ env.WORKFLOW_ID }}/dispatches -d '{"ref":"${{ env.PARENT_BRANCH }}", "inputs": { "stage":"${{ env.STAGE }}", "component":"${{ env.COMPONENT }}", "image_tag":"${{ env.TAG }}", "commit_author":"${{ env.AUTHOR }}", "commit_message":"${{ env.COMMIT_MESSAGE }}" }}'
```

Comment:
It looks like gha has a `toJSON` function that can escape these values properly for substitution into json. Can you use that? Otherwise shell/json metacharacters will mess things up

Reply:
removed the quotes completely https://github.com/temporalio/docker-builds/pull/14/files#diff-d06ee8a9a91c214c65420a8f1f4243c90f1471c1655fc080bd13aad99410a987R62